colorschemes/dark-spook.conf

```diff
@@ -0,0 +1,117 @@
+#
+# Copyright 2012 Campbell Barton <ideasman42(at)gmail(dot)com>
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Library General Public
+# License as published by the Free Software Foundation; either
+# version 2 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Library General Public License for more details.
+#
+# You should have received a copy of the GNU Library General Public
+# License along with this library; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place - Suite 330,
+# Boston, MA 02111-1307, USA.
+#
+
+[theme_info]
+name=DarkSpook
+description=Based on the InkPot theme of the same name.
+# incremented automatically, do not change manually
+version=1
+author=Vukić Srđan <tux.lector(at)gmail(dot)com>
+url=https://hardcoder.xyz
+# list of each compatible Geany release version
+compat=1.22;1.23;1.23.1;1.24;1.25;1.26;1.27;1.28;1.29;1.33;1.38;2.0
+
+[named_styles]
+default=#626CAD;#15181D;false;false
+error=#542929;#121C2B;false;false
+
+# Editor styles
+#-------------------------------------------------------------------------------
+
+selection=;#4e4e8f;false;false
+current_line=#000;#2d2d32;;false
+brace_good=#cfbfad;#4e4e8f;false;false
+brace_bad=#cfbfad;#f00;false;false
+margin_line_number=#8b8bcd;#2e2e2e;false;false
+margin_folding=#000;#3e3e3e;false;false
+fold_symbol_highlight=#6e6e6e;;false;false
+indent_guide=#3b3b4d;;false;false
+caret=#8b8bff;#505050;false;false
+marker_line=#000;#ff0;false;false
+marker_search=#000;#b8f4b8;false;false
+marker_mark=#000;#b8f4b8;false;false
+call_tips=default
+white_space=indent_guide
+
+# Generic programming languages
+#-------------------------------------------------------------------------------
+
+comment=#3A404A;;false;false
+comment_doc=#2F3742;#1A1E24;false;true
+comment_line=comment
+comment_line_doc=#2F3742;;true;true
+comment_doc_keyword=#3A404A;;false;true
+comment_doc_keyword_error=comment_doc
+
+number=#B4469D;;true;false
+number_1=#AD46B4;;true;false
+number_2=#27B52B;;true;false
+
+type=#2B4D45;;true;true
+class=#AEBEFF;;true;true
+function=#18678C;;true;true
+parameter=#188C31;;true;true
+
+keyword=#B3895D;;false;true
+keyword_1=#3D5070;;true;false
+keyword_2=#946767;;false;true
+keyword_3=#B5B5B5;;false;true
+keyword_4=#7F7F7F;;false;true
+
+identifier=#3CCA62;;false;true
+identifier_1=#507875;;false;true
+identifier_2=#457C54;;false;true
+identifier_3=#256536;;false;true
+identifier_4=#8BBC98;;false;true
+
+string=#ffcd8b;;false;false
+string_1=#BFAC91;#1C1F26;false;false
+string_2=#BA9665;#1C1F26;false;false
+string_3=#CDA7A1;;false;false
+string_4=#A1CDCA;;false;false
+string_eol=#A1C1CD;;false;false
+character=#A86027;;false;false
+backticks=#008C39l;;false;false
```

Comment:
Geany complains about the invalid color value:
```
Geany-INFO: 00:00:55.142: Bad color '#008C39l'
Geany-INFO: 00:00:55.145: selection style is set to invisible - ignoring!
```
Note also the selection style warning.

Reply:
> Geany complains about the invalid color value:

Ouch! 
Thanks. 

That 7th `l` character (not including `#` ) is actually ';', but my finger apparently slipped. 
Will remove it. 

> Note also the selection style warning.

Yes, that I am aware of. Tried various combinations, it is quite ok like this. 

Now, I noticed that the name of the file is dark-spook.conf, but Titled Name is DarkSpook.   
Should I correct that as well ? Into `Dark Spook` from `DarkSpook` or I leave as is ?

Reply:
> > Geany complains about the invalid color value:
> 
> Ouch! Thanks.
> 
> That 7th `l` character (not including `#` ) is actually `;`, but my finger apparently slipped. Will remove it.

:)

> > Note also the selection style warning.
> 
> Yes, that I am aware of. Tried various combinations, it looks quite ok like this.

It would be nice if we could avoid the warning.

> I noticed that the name of the file is `dark-spook.conf`, but Titled Name is `DarkSpook`. Should I correct that as well ? From `DarkSpook` to `Dark Spook` into config or I leave as is ?

I think this is OK, some other themes do this as well.